src/pages/_app.tsx

```diff
@@ -33,6 +34,7 @@ export default function App({ Component, pageProps }: AppPropsWithLayout) {
   useBosLoaderInitializer();
   useHashUrlBackwardsCompatibility();
   usePageAnalytics();
+  useClickTracking();
```

Comment:
Since you're calling the hook globally in `app.tsx`, this should be the only place the hook is used I believe since it will affect the entire application. The other pages where you also call `useClickTracking()` would result in duplicate analytics click events being fired off, right?